server/helpers/jwt.js

```diff
@@ -1,35 +1,44 @@
+/* eslint-disable import/no-unresolved */
```

Comment:
Why is this here?

Reply:
Because this is having a redline even though it resolves appropriately, and I don't disable this rule for the entire project; hence, I didn't add it to Eslint rules